### PR TITLE
changes to upgrade user-mla loki to v2.9.2 via helm chart v0.78.0

### LINKED ISSUE #12921: [mla] upgrade user cluster MLA components
### Description of the feature you would like to add / User story

User cluster MLA components require some updates to improve features and performance of KKP MLA stack.

### Implementation

- [x] upgrade cortex to TBD (PR: https://github.com/kubermatic/kubermatic/pull/12935)
- [x] upgrade grafana to TBD
- [x] upgrade loki to TBD
- [x] upgrade minio to TBD

### PR DESCRIPTION
**Pull request by dharapvj**
**What this PR does / why we need it**:
User-MLA stack needs an upgrade. This PR brings upgrade of `loki-distributed` component of user-mla stack.

**Which issue(s) this PR fixes**:

Partially Fixes #12921

**What type of PR is this?**

/kind chore

**Special notes for your reviewer**:
Entire `values.yaml` file loki-distributed chart has been cleaned up to only retain kubermatic's customizations. This way, it will be easier to bring in upstream upgrades in future.

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
action required: [User-mla] If you had copied `values.yaml` of loki-distributed chart to further customize it, then please cleanup your copy of `values.yaml` for user-mla to retain your customization only.
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by wurbanski** [comment]
@dharapvj does this need a similar installer work as cortex did?

**Comment by dharapvj** [comment]
No... there are no known properties that got removed etc.

In normal circumstances - end user will know even know that loki got upgraded.

But again - if someone has copied the entire values.yaml from our old chart, and then customized it, there could be unpredictable outcome since the new chart has `config` provided as "string" whereas in old chart it was a regular yaml dict. Upstream values.yaml has provided new option called `structuredConfig` to provide yaml based config overrides.

You can see in the changes that we have updated our values.yaml to use `structuredConfig` now so that it is easier to provide overridden bits there.

In summary - I am not sure if we can detect this and validate via installer here.

Hope that helps.

**Comment by wurbanski** [comment]
/lgtm
/approve

@dharapvj I think you can simplify the release notes by saying that structuredConfig is now to be used instead of config field?
